### ISSUE TITLE
pylint manpage refers to nonexistent /usr/share/doc/pythonX.Y-pylint/

### ISSUE THREAD

**Issue by sandrotosi** [PyCQA/pylint#1508]
Hello,
this was reported to Debian as http://bugs.debian.org/825187

```
pylint(1) manpage reads:

SEE ALSO
      /usr/share/doc/pythonX.Y-pylint/

but no such directories exist.
```

**Comment by rogalski** [comment]
Thanks for submitting an issue!

I guess we have an underlying problem of not being up-to-date with manpage at all. In my opinion, ideally we'd want to have it generated automatically based on built-in Pylint help before a release, or maybe get rid of manpage completely.

**Comment by sandrotosi** [comment]
i like the idea of updating the manpage from the running code (ie --help) not so much about getting rid of the manpage entirely

**Comment by rogalski** [comment]
As it turned out, we already have manpage generator - `pylint --generate-man`.

I'm not sure what would be a standard practise in community.

Is it possible to regenerate man-page during generation of `*.deb`? In this way we as pylint maintainers provide a way to generate up-to-date docs, you as packager are responsible for providing complete package for given platform.

**Comment by sandrotosi** [comment]
i could generate the manpage for `pylint` using the `--generate-man` switch, but what about the other utilities?  pyreverse, symilar etc?

**Comment by DanielNoord** [comment]
@rogalski @sandrotosi I just went through some old issues and stumbled upon this.

Is there anything we can do from the `pylint` side to resolve this? (Or has this been fixed in the four years since this issue was opened) Or is this something that needs to be fixed at the `debian` side?

**Comment by sandrotosi** [comment]
i thought pylint removed the manpages from the repo?

**Comment by Pierre-Sassoulas** [comment]
Yes we did because it was very outdated. @sandrotosi, I think you generate it when packaging for Debian ? I suppose you're using the generate-man command ?

**Comment by sandrotosi** [comment]
> Yes we did because it was very outdated. @sandrotosi, I think you generate it when packaging for Debian ? I suppose you're using the generate-man command ?

we are currently not running the `generate-man` command

**Comment by DanielNoord** [comment]
I think we can then close this issue as it no longer relevant? We're not hosting the manpages any longer.
